gulpfile.js

```diff
@@ -107,22 +107,16 @@ const appBundles = [
 //  - Add the relevant dependencies to the project
 //  - Create an entry point in `src/shared/polyfills/{set}` and a feature
 //    detection function in `src/shared/polyfills/index.js`
-//  - Add an entry to the list below to generate the polyfill bundle
 //  - Add the polyfill set name to the required dependencies for the parts of
```

Comment:
Trivial aside: I find the term "set" (very mildly) confusing throughout here, and most specifically as a variable name in one of the `map`-pings. Mostly because `set` is, if memory serves, literally the most overloaded word in the English language :). "collection"  or "bundle" might work. Or just leave it because it's really not a crisis.

Reply:
I see your point. I'd prefer to leave as-is in this PR since the term "polyfill set" is used in a few places but I'm open to revising later.